packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3740,9 +3800,16 @@ export default class Meeting extends StatelessWebexPlugin {
               LoggerProxy.logger.info('Meeting:index#join --> enabled to recieve transcription!');
             }
           }
+          // @ts-ignore - config coming from registerPlugin
+          if (this.config.receiveReaction || options.receiveReaction) {
+            if (this.isReactionsSupported()) {
+              await this.receiveReaction();
```

Comment:
I don't think we need to await. Also receiveReaction doesn't need to be async.
Also, maybe call this method receiveReaction`s`

Reply:
oki, should I remove the async from the function declaration (line :3589)